web-nfc/NFCReader.https.html

```diff
@@ -131,4 +131,47 @@
   await promise;
 }, "Test that NFCReader.scan succeeds if NFCScanOptions.url is empty.");
 
+nfc_test(async (t, mockNFC) => {
```

Comment:
You can also test an invalid AbortSignal.

Reply:
Done.